p2p/switch_test.go

```diff
@@ -348,7 +348,7 @@ func TestSwitchStopsNonPersistentPeerOnError(t *testing.T) {
 }
 
 func TestSwitchStopPeerForError(t *testing.T) {
-	s := httptest.NewServer(stdprometheus.UninstrumentedHandler())
+	s := httptest.NewServer(promhttp.Handler())
```

Comment:
why change?

Reply:
SA1019: stdprometheus.UninstrumentedHandler is deprecated: